modules/index.js

```diff
@@ -122,10 +122,20 @@ function addNavigationTracking() {
     } else {
       sampleRUM('enter', payload); // enter site
     }
+    fflags.enabled('redirect', () => {
+      const from = new URLSearchParams(window.location.search).get('redirect-from');
+      if (redirectCount || from) {
+        sampleRUM('redirect', { source: from, target: redirectCount || 1 });
```

Comment:
```suggestion
        sampleRUM('redirect', { source: from || '', target: redirectCount || 1 });
```

`from` might be null